INSTALL.md

```diff
@@ -26,6 +26,7 @@ targets.
  - Modern C++ compiler that supports `-std=c++14`
    - clang++ 3.5 or later _or_
    - g++-5 or later
+ - CMake (for build only)
  - Python 2.x (for build only)
  - [Node.js](https://nodejs.org/) (for build only)
  - [`pkg-config`](https://wiki.freedesktop.org/www/Software/pkg-config/) (for build only)
```

Comment:
created ticket for Python removal: https://github.com/mapbox/mapbox-gl-native/issues/6010